docs/guides/cli.md

```diff
@@ -61,6 +62,65 @@ Connection: keep-alive
 
 The body, headers, etc. for this response will be taken from the API description document.
 
+#### Request Specific Examples
+
+You can request a specific example from your document by using the `__example` query string parameter.
+
+```bash
+curl -v http://127.0.0.1:4010/pets/123?__example=exampleKey
+```
+
+#### Dynamic Response
+
+You can override the `--dynamic|-d` CLI param (which decides whether the generated example is static or dynamic) through the `__dynamic` query string parameter.
+
+```bash
+curl -v http://127.0.0.1:4010/pets/123?__dynamic=false
+```
+
+## Proxy
+
+This command creates an HTTP server that will proxy all the requests to the specified upstream server. Prism will analyze the request coming in and the response coming back from the upstream server and report the discrepancies with what's declared in the provided OpenAPI document.
+
+**Note:** Currently the proxy command _expects_ a JSON Payload for both the request and the response. If the response's `content-type` header does not indicate a JSON-ish payload (`application/json`, `application/*+json`), then the payload will be just returned as text and send back to the original client.
+
+```
+prism proxy examples/petstore.oas2.yaml https://petstore.swagger.io/v2
+
+[CLI] …  awaiting  Starting Prism…
+[HTTP SERVER] ℹ  info      Server listening at http://127.0.0.1:4010
+[CLI] ●  note      GET        http://127.0.0.1:4010/pets
+[CLI] ●  note      POST       http://127.0.0.1:4010/pets
+[CLI] ●  note      GET        http://127.0.0.1:4010/pets/10
+```
+
+The output violations will be reported on the standard output and as a response header, following the [Warning spec](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Warning)
+
+```bash
+prism proxy examples/petstore.oas2.yaml https://petstore.swagger.io/v2 --report-violations httpHeader
+… …
+
+curl -v -s http://localhost:4010/pets/10 > /dev/null
+
+< HTTP/1.1 404 Not Found
+< warning: body should be object
+```
+
+You can see there's a `warning` header which is a JSON object with all the violations found in the response.
+
+```bash
+prism proxy examples/petstore.oas2.yaml https://petstore.swagger.io/v2 --errors
+… …
+
+curl -v http://localhost:4010/pets/10
+
+< HTTP/1.1 422 Unprocessable Entity
+```
+
+The response body contains the found output violations.
```

Comment:
Let’s show them how it looks in case they’re a visual learner and need to see what the JSON looks like in order for this to click.

Reply:
Done